lms/static/js/bookmarks/views/bookmarks_list.js

```diff
@@ -60,7 +60,17 @@
             },
 
             visitBookmark: function (event) {
-                window.location = event.target.pathname;
+                var bookmark_id = event.currentTarget.dataset.id;
+                var component_usage_id = bookmark_id.split(',')[1];
+                Logger.log(
```

Comment:
Can you use this pattern: https://github.com/edx/edx-platform/blob/master/lms/static/js/search/base/views/search_item_view.js#L42-L69 ?